### ISSUE TITLE
Update Delay Packets Mod?

### ISSUE THREAD

**Issue by Equinox-76** [TheBosoYolo/delay-packets#4]
Could you update your delay packets mod to **1.20 - 1.20.4**? The mod has been **very** helpful **=D**.

**Comment by pernydev** [comment]
For sure! Right now I would really need this for 1.21!

**Comment by hecker124dev** [comment]
yeah it would be really cool im tryna make an update for this but i dont have an idea how to code
and it should be updated to 1.21.1

**Comment by AgentKosticka** [comment]
I'm so sorry, I will mark this repository as archived. I have anabandoned the project since the Clone feature kept crashing with no apparent reason as to why when i tried to update, and it was exhausting to repeatedly read through hundreds if not thousands of lines to figure out which packets needed to be captured (without using eg.: c#'s Reflection)
